x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form.tsx

```diff
@@ -827,44 +828,59 @@ export const RuleForm = ({
       )}
 
       <EuiFlexItem>
-        <EuiFormRow fullWidth data-test-subj="alertDelayFormRow" display="rowCompressed">
-          <EuiFieldNumber
-            fullWidth
-            min={1}
-            value={alertDelay || ''}
-            name="alertDelay"
-            data-test-subj="alertDelayInput"
-            prepend={[
-              i18n.translate('xpack.triggersActionsUI.sections.ruleForm.alertDelayFieldLabel', {
-                defaultMessage: 'Alert after',
-              }),
-              <EuiIconTip
-                position="right"
-                type="questionInCircle"
-                content={
-                  <FormattedMessage
-                    id="xpack.triggersActionsUI.sections.ruleForm.alertDelayFieldHelp"
-                    defaultMessage="An alert occurs only when the specified number of consecutive runs meet the rule conditions."
-                  />
+        <EuiAccordion
+          id="alertSettingsAccordion"
+          data-test-subj="alertSettingsAccordion"
+          buttonContent={
+            <EuiTitle size="xxs">
```

Comment:
I don't think this should be as prominent / use EuiTitle.  Let's use `EuiText size="s"` instead.  

Unless of course I'm missing a current example that we're following...

Reply:
That is my bad! Updated here [2246dbd](https://github.com/elastic/kibana/pull/177095/commits/2246dbdf2bf32d085ca17fc8c24e0b0313388ded)
<img width="584" alt="Screen Shot 2024-02-21 at 9 17 10 AM" src="https://github.com/elastic/kibana/assets/109488926/ddb9a570-5b0b-47f8-9b13-4a6675305a0e">